applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_adjoint_solver.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+# importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as KCPFApp
+from KratosMultiphysics.CompressiblePotentialFlowApplication.potential_flow_solver import PotentialFlowSolver
+import KratosMultiphysics.StructuralMechanicsApplication
```

Comment:
```suggestion
from KratosMultiphysics.StructuralMechanicsApplication import ReplaceMultipleElementsAndConditionsProcess
```
this way it would be clear that only this is used from StructuralMechanics